@stianst - figured out that 

> mvn -am -pl docs/antora-ccutil clean install -DskipTests -DskipAccount2

doesn't work as expected if there hasn't been a previous full build, will fail with 

> [ERROR] Failed to execute goal io.quarkus:quarkus-bootstrap-maven-plugin:2.13.6.Final:extension-descriptor (default) on project keycloak-quarkus-server: Deployment artifact org.keycloak:keycloak-quarkus-server-deployment::jar:999-SNAPSHOT is missing the following dependencies from its configuration: org.keycloak:keycloak-quarkus-server::jar, io.quarkus:quarkus-vertx-http-deployment::jar, io.quarkus:quarkus-mutiny-deployment::jar, io.quarkus:quarkus-smallrye-context-propagation-deployment::jar, io.quarkus:quarkus-vertx-deployment::jar, io.quarkus:quarkus-arc-deployment::jar, io.quarkus:quarkus-netty-deployment::jar, io.quarkus:quarkus-reactive-routes-deployment::jar, io.quarkus:quarkus-jackson-deployment::jar, io.quarkus:quarkus-resteasy-deployment::jar, io.quarkus:quarkus-resteasy-server-common-deployment::jar, io.quarkus:quarkus-resteasy-common-deployment::jar, io.quarkus:quarkus-resteasy-jackson-deployment::jar, io.quarkus:quarkus-hibernate-orm-deployment::jar, io.quarkus:quarkus-narayana-jta-deployment::jar, io.quarkus:quarkus-caffeine-deployment::jar, io.quarkus:quarkus-agroal-deployment::jar, io.quarkus:quarkus-datasource-deployment::jar, io.quarkus:quarkus-jdbc-h2-deployment::jar, io.quarkus:quarkus-jdbc-postgresql-deployment::jar, io.quarkus:quarkus-jdbc-mariadb-deployment::jar, io.quarkus:quarkus-jdbc-mysql-deployment::jar, io.quarkus:quarkus-jdbc-mssql-deployment::jar, io.quarkus:quarkus-jdbc-oracle-deployment::jar, io.quarkus:quarkus-core-deployment::jar, io.quarkus:quarkus-smallrye-health-deployment::jar, io.quarkus:quarkus-jsonp-deployment::jar, io.quarkus:quarkus-micrometer-deployment::jar, io.quarkus:quarkus-micrometer-registry-prometheus-deployment::jar, io.quarkus:quarkus-logging-json-deployment::jar, io.quarkus:quarkus-logging-gelf-deployment::jar -> [Help 1]

Updating the docs to use a full build without `-am -pl ...`. The cause needs to be investigated in a separate issue.

--- a/docs/guides/src/main/server/configuration-production.adoc
+++ b/docs/guides/src/main/server/configuration-production.adoc
@@ -22,7 +22,7 @@ In a production environment, Keycloak instances usually run in a private network
 For details on the endpoint categories and instructions on how to configure the public hostname for them, see the <@links.server id="hostname"/> guide.
 
 == Reverse proxy in a distributed environment
-Apart from <<Setting the hostname for Keycloak>>, production environments usually include a reverse proxy / load balancer component. It separates and unifies access to the network used by your company or organization. For a Keycloak production environment, this component is recommended.
+Apart from <@links.server id="hostname"/>, production environments usually include a reverse proxy / load balancer component. It separates and unifies access to the network used by your company or organization. For a Keycloak production environment, this component is recommended.
 
 For details on configuring proxy communication modes in Keycloak, see the <@links.server id="reverseproxy"/> guide. That guide also recommends which paths should be hidden from public access and which paths should be exposed so that Keycloak can secure your applications.
 

--- a/docs/guides/src/main/templates/links.adoc
+++ b/docs/guides/src/main/templates/links.adoc
@@ -1,3 +1,1 @@
-<#macro server id>
-link:{links_server_${id}_url}[{links_server_${id}_name}]
-</#macro>
+<#macro server id>link:{links_server_${id}_url}[{links_server_${id}_name}]</#macro>

